lib/Utility/Utils.dart

```diff
@@ -0,0 +1,21 @@
+// create Utils class
+import 'dart:js_interop';
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:js_util' as js_util;
```

Comment:
`js_util`shouldn't be used
see: https://dart.dev/interop/js-interop

Reply:
After some investigations, I think it's too premature to expect it to be WASM compliant... We'll put in TODOs when we put in js_util to fix things later. Let's keep building